src/content/cookbook/lists/floating-app-bar.md

```diff
@@ -105,28 +139,66 @@ CustomScrollView(
 :::tip
 Play around with the
 [various properties you can pass to the `SliverAppBar` widget][],
-and use hot reload to see the results. For example, use an `Image`
-widget for the `flexibleSpace` property to create a background image that
-shrinks in size as it's scrolled offscreen.
+and use hot reload to see the results. For example, use an
+`Image` widget for the `flexibleSpace` property to create a
+background image that shrinks in size as it's scrolled
+offscreen.
 :::
 
+{% endtab %}
 
-### 3. Add a list of items using a `SliverList`
+{% tab "Cupertino widgets" %}
 
-Now that you have the app bar in place, add a list of items to the
-`CustomScrollView`. You have two options: a [`SliverList`][]
-or a [`SliverGrid`][].  If you need to display a list of items one after the other,
-use the `SliverList` widget.
-If you need to display a grid list, use the `SliverGrid` widget.
+Flutter provides the [`CupertinoSliverNavigationBar`][]
+widget, which you can use to add a navigation bar for your
```

Comment:
Should `for` be `to` here? That seems to flow better for me verbally.

```suggestion
widget, which you can use to add a navigation bar to your
```

Reply:
Updated this section. Can you take another look?